sgtool/file.go

```diff
@@ -28,6 +28,10 @@ const (
 	TarGz
 )
 
+const (
+	DARWIN = "darwin"
```

Comment:
This should say `Darwin`?

Reply:
Since Go constants are not all upper case

Reply:
Thank you! Switching between TS and Go can be confusing 😅  Fixed!